terraform/testnet/forge/files/cleanup_forge.sh

```diff
@@ -3,6 +3,4 @@
 # Script to be run in a Forge pod after the test runs as cleanup
 # This separates the pod lifecycle from aptos/aptos
 
-# CoreDNS in EKS needs to scale down to avoid cluster resource contention
-# after the cluster has been scaled down by forge test cleanup
-kubectl scale deployment/coredns --namespace kube-system --replicas=2
+echo "Forge clean"
```

Comment:
so the cleanup script doesn't do anything?

Reply:
Yea nothing at the moment. Left it here so it's easy to add things in the future, since it's still pretty useful capability